egs/wsj/s5/utils/format_lm_sri.sh

```diff
@@ -104,6 +104,27 @@ awk '{print $1}' $out_dir/words.txt > $tmpdir/voc || exit 1;
 change-lm-vocab -vocab $tmpdir/voc -lm $tmpdir/lm.gz -write-lm $tmpdir/out_lm \
   $srilm_opts || exit 1;
 
+# change-lm-vocab introduces error in out_lm when lm.gz has an order
```

Comment:
Actually I was wrong on the error message (which is you're right a warning). The error is generated latter in the code: 
ERROR (arpa2fst:Read():arpa-file-parser.cc:228) in line 3235: Invalid or unexpected directive line '\3-grams:', expected \end\.

Here is the full output of the format_lm_sri script:

> Converting '/home/mbernard/dev/abkhazia/test/language/s5/data/local/language/G.arpa.gz' to FST
> arpa2fst /tmp/kaldi.J7H6/out_lm
> LOG (arpa2fst:Read():arpa-file-parser.cc:90) Reading \data\ section.
> LOG (arpa2fst:Read():arpa-file-parser.cc:147) Reading \1-grams: section.
> LOG (arpa2fst:Read():arpa-file-parser.cc:147) Reading \2-grams: section.
> WARNING (arpa2fst:Read():arpa-file-parser.cc:134) Zero ngram count in ngram order 3(look for 'ngram 3=0' in the \data\  section). There is possibly a problem with the file.
> ERROR (arpa2fst:Read():arpa-file-parser.cc:228) in line 3235: Invalid or unexpected directive line '\3-grams:', expected \end\.
> ERROR (arpa2fst:Read():arpa-file-parser.cc:228) in line 3235: Invalid or unexpected directive line '\3-grams:', expected \end\.
> [stack trace: ]
> kaldi::KaldiGetStackTrace()
> kaldi::KaldiErrorMessage::~KaldiErrorMessage()
> kaldi::ArpaFileParser::Read(std::istream&, bool)
> arpa2fst(main+0x678) [0x40f298]
> /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf5) [0x7f68dd512b45]
> arpa2fst() [0x410282]
> ERROR: FstHeader::Read: Bad FST header: standard input
> fstisstochastic /tmp/tmprYKXjl/G.fst
> 0 0
> Succeeded in formatting LM '/home/mbernard/dev/abkhazia/test/language/s5/data/local/language/G.arpa.gz' -> '/tmp/tmprYKXjl/G.fst'